tests/es6-timed-map.spec.ts

```diff
@@ -168,13 +168,36 @@ describe('Es6TimedMap', () => {
   });
 
   // "time" based functions
-  describe('timers', () => {
-    test.todo('exists');
-    test.todo('returns timers in insertion order');
-    test.todo('returns timers in expiration order');
-    test.todo('returns timers in expiration order');
+  describe.only('timers', () => {
```

Comment:
```suggestion
  describe('timers', () => {
```

Don't forget to remove the `.only`

Reply:
Removed